app/javascript/gobierto_data/lib/sql-theme.css

```diff
@@ -0,0 +1,21 @@
+.sql {
```

Comment:
Too simple name

Reply:
This class is the codemirror. I only use it to styling autocomplete